scripts/sct_register_multimodal.py

```diff
@@ -191,7 +191,7 @@ def __init__(self):
 
 # Parameters for registration
 class Paramreg(object):
-    def __init__(self, step='1', type='', algo='syn', metric='MeanSquares', iter='10', shrink='1', smooth='0', gradStep='0.5', init='', poly='5', slicewise='0', laplacian='0', dof='Tx_Ty_Tz_Rx_Ry_Rz', smoothWarpXY='2', pca_eigenratio_th='1.6'):
+    def __init__(self, step='', type='', algo='syn', metric='MeanSquares', iter='10', shrink='1', smooth='0', gradStep='0.5', init='', poly='5', slicewise='0', laplacian='0', dof='Tx_Ty_Tz_Rx_Ry_Rz', smoothWarpXY='2', pca_eigenratio_th='1.6'):
```

Comment:
We actually need a default value to be able to initialize a Paramreg object before actually setting its arguments. (see line 237). I removed it on commit d090872 and put it back on commit 3f32135.